wpilibc/src/main/native/include/frc/PneumaticHub.h

```diff
@@ -157,6 +157,15 @@ class PneumaticHub : public PneumaticsBase {
     uint32_t Brownout : 1;
     uint32_t CanWarning : 1;
     uint32_t HardwareFault : 1;
+
+    /**
+     * Gets whether there is a fault at a specified channel.
+     * @param channelIndex Index to check for faults.
+     * @return If there is a fault at the specified index.
+     * @throws A ChannelOutOfIndex error if the given int is outside of the
+     * range supported by the hardware.
+     */
+    bool GetChannelFaultAtIndex(int channelIndex) const;
```

Comment:
Should this be called `GetChannelFault(int channel)` instead? We don't refer to "channel indexes" anywhere else - the channel _is_ the index.

Reply:
Agreed.